### PR TITLE
Add docs about tile size variants

### PR DESCRIPTION
**Pull request by rmarianski**
This should only get merged once the buffered terrain service is in production, and we are ready to go public with it.

### CODE REVIEW

**Comment by rmarianski** [comment]
@nvkelso I was pretty terse with what I added. Could you take a quick look and add in some more details as you see fit?

File: docs/use-service.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Get started with Mapzen Terrain Tiles
 
-The Mapzen terrain tiles provide basemap elevation coverage of the world in a raster tile format. Tiles are available for zooms 0 through 15 and are available in several spatial data formats including PNG and GeoTIFF. The tiles also can be in a raw elevation and processed normal value format that's optimized for mobile and web display, and desktop analytical use. Data is available in both Web Mercator (EPSG:3857) projected and raw latlng. Learn more about the [various data formats](fileformats.md) offered.
+The Mapzen terrain tiles provide basemap elevation coverage of the world in a raster tile format. Tiles are available for zooms 0 through 15 and are available in several spatial data formats including PNG and GeoTIFF. The tiles also can be in a raw elevation and processed normal value format that's optimized for mobile and web display, and desktop analytical use. Data is available in both Web Mercator (EPSG:3857) projected and raw latlng. Learn more about the [various data formats](formats.md) offered.
```
Comment (nvkelso) [inline docs/use-service.md]: Note: this URL change is a bug fix to the correct link name so no url forwarding is needed.

**Comment by nvkelso** [comment]
This change won't show up on mapzen.com/documentation until we do a release of this repo:

- https://github.com/mapzen/documentation/blob/master/Makefile#L6

So we can merge this as-is now.

File: docs/use-service.md [hunk 2]
```diff
@@ -14,10 +14,20 @@ You can request tiles using Mapzen's global CDN:
 
   `https://tile.mapzen.com/mapzen/terrain/v1/terrarium/{z}/{x}/{y}.png`
 
+Supported tile size variants:
```
Comment (nvkelso) [inline docs/use-service.md]: We should indicate these are _optional_.

File: docs/use-service.md [hunk 3]
```diff
@@ -14,10 +14,20 @@ You can request tiles using Mapzen's global CDN:
...
+
+  `https://tile.mapzen.com/mapzen/terrain/v1/{256,512,260,516}/terrarium/{z}/{x}/{y}.png`
+
+The 260 and 516 variants contain 2 pixels of buffering on each edge.
+
 ##### Normal
 
   `https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png`
 
+Supported tile size variants:
```
Comment (nvkelso) [inline docs/use-service.md]: We should indicate these are _optional_ and the default when not specified is 256.

**Comment by nvkelso** [comment]
@rmarianski can you also add documentation in the vector-datasource repo about the _optional_ 256 and 512 sizes?

File: docs/use-service.md (hunk 2 above)
Comment (rmarianski) [inline docs/use-service.md]: Updated in e4f3a3f

File: docs/use-service.md (hunk 3 above)
Comment (rmarianski) [inline docs/use-service.md]: Updated in e4f3a3f

**Comment by rmarianski** [comment]
> @rmarianski can you also add documentation in the vector-datasource repo about the optional 256 and 512 sizes?

Just double checking, you mean just for the tile sizes, and nothing to do with terrain right?

**Pull request merged**